airflow/providers/google/cloud/operators/dataform.py

```diff
@@ -348,6 +349,81 @@ def execute(self, context: Context):
         return WorkflowInvocation.to_dict(result)
 
 
+class DataformQueryWorkflowInvocationOperator(GoogleCloudBaseOperator):
+    """
+    Returns WorkflowInvocationActions in a given WorkflowInvocation.
+
+    :param project_id: Required. The ID of the Google Cloud project that the task belongs to.
+    :param region: Required. The ID of the Google Cloud region that the task belongs to.
+    :param repository_id: Required. The ID of the Dataform repository that the task belongs to.
+    :param workflow_invocation_id:  the workflow invocation resource's id.
+    :param retry: Designation of what errors, if any, should be retried.
+    :param timeout: The timeout for this request.
+    :param metadata: Strings which should be sent along with the request as metadata.
+    :param gcp_conn_id: The connection ID to use when fetching connection info.
+    :param impersonation_chain: Optional service account to impersonate using short-term
+        credentials, or chained list of accounts required to get the access_token
+        of the last account in the list, which will be impersonated in the request.
+        If set as a string, the account must grant the originating account
+        the Service Account Token Creator IAM role.
+        If set as a sequence, the identities from the list must grant
+        Service Account Token Creator IAM role to the directly preceding identity, with first
+        account from the list granting this role to the originating account (templated).
+    """
+
+    template_fields = (
+        "project_id",
+        "region",
+        "repository_id",
+        "workflow_invocation_id",
+        "impersonation_chain",
+    )
+    operator_extra_links = (DataformWorkflowInvocationLink(),)
```

Comment:
The link is declared, but never used.
Please, add a call for `DataformWorkflowInvocationLink.persist()` in the `execute()` method. It will make Airflow to generate a button in UI that references the URL built by the link class.
BTW, is it possible to see actions list in the UI? If so, then it makes sense to declare a new link class and use it here instead.

Reply:
Added the call for persist() in execute() method.